src/Akeneo/Component/StorageUtils/Cursor/CursorInterface.php

```diff
@@ -9,6 +9,6 @@
  * @copyright 2015 Akeneo SAS (http://www.akeneo.com)
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  */
-interface CursorInterface extends \Countable, \Iterator
```

Comment:
in fact, I removed it because I thought it was not used in the application as there was an inconsistence between mongo & orm (mongo returned all items, orm only current items), but it's used, there are a lot of specs broken
so I revert my commit